packages/components/src/components/hds/advanced-table/td.hbs

```diff
@@ -8,12 +8,14 @@
   aria-rowspan={{@rowspan}}
   aria-colspan={{@colspan}}
   {{style grid-row=this.rowspan grid-column=this.colspan}}
-  {{did-insert this.didInsert}}
-  {{will-destroy this.willDestroy}}
+  {{hds-advanced-table-cell
+    handleEnableFocusTrap=this.enableFocusTrap
+    shouldTrapFocus=this._shouldTrapFocus
+    setCellElement=this.setElement
+  }}
   {{! @glint-expect-error - https://github.com/josemarluedke/ember-focus-trap/issues/86 }}
   {{focus-trap
     isActive=this._shouldTrapFocus
-    shouldSelfFocus=true
```

Comment:
removed `shouldSelfFocus` because it doesn't do anything when initialFocus is set (see within focusTrapOptions)